generator-foodq/generators/app/index.js

```diff
@@ -261,12 +259,10 @@ module.exports = class extends Generator {
 
   writing() {
     this.log('in writing');
-    this.destinationRoot(path.join(this.destinationRoot(), this.answers.food));
+    this.destinationRoot(path.join(this.destinationRoot(), _.get(this, "answers.food", "")));
     this.log('destinationRoot: ' + this.destinationRoot());
-    this.fs.copyTpl(
-      this.templatePath('index.html'),
-      this.destinationPath('public/index.html'),
-      {
+    this.fs.copyTpl(this.templatePath('index.html'),
+    this.destinationPath('public/index.html'), {
```

Comment:
this.fs.copyTpl(this.templatePath('index.html'), this.destinationPath('public/index.html'), {